packages/wonder-blocks-form/components/radio-core.js

```diff
@@ -59,6 +59,7 @@ const StyledInput = addStyle("input");
                 <StyledInput
                     {...sharedProps}
                     type="radio"
+                    aria-invalid={error}
                     aria-checked={checked}
```

Comment:
Could you please remove `aria-checked` from RadioButtons as well?

Reply:
This reminds me of that other PR a while back.  It seems like if there's already an attribute for checked or disabled or whatever we don't need to add the `aria-` version of it.  It's too bad there isn't way to check for this automatically.